src/devices/src/virtio/block/io/async_io.rs

```diff
@@ -118,18 +118,21 @@ impl<T> AsyncFileEngine<T> {
 
         let wrapped_user_data = WrappedUserData::new_with_dirty_tracking(addr, user_data);
 
-        self.ring
-            .push(Operation::read(
+        // Safe because we trust that the host kernel will us pass back a completed entry with this
```

Comment:
Seems like there's a topic issue: `will us pass back`. It's present in other comments as well.

Reply:
done, copy paste error 😄